code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -25,7 +25,7 @@
 		weapon_edge = 0
 
 	hit_impact(effective_force, get_step(user, src))
-	damage_through_armor(effective_force, I.damtype, hit_zone, ARMOR_MELEE, armour_pen = I.armor_penetration + (user.stats.getStat(STAT_ROB) * 0.25), used_weapon = I, sharp = weapon_sharp, edge = weapon_edge, post_pen_mult = I.post_penetration_dammult)
```

Comment:
melee rob nerf V_V it should still scale a little imo

Reply:
Probably, that got removed in the process of just changing over old->new code.